curiouslearning/FeedTheMonsterJS
# fix: feedback audio issue in levelendscene (FM-341)

# Changes
- feedback audio issue in levelendscene

# How to test
- Open the start screen of FTM.
- Tap or click any part of the start screen to proceed to the next screen
- Select any level and play the game
- Make sure the feedback audio dont overlap to levelend scene

Ref: [FM-341](https://curiouslearning.atlassian.net/browse/FM-341)


[FM-341]: https://curiouslearning.atlassian.net/browse/FM-341?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
